webapp-django/crashstats/crashstats/decorators.py

```diff
@@ -95,9 +95,6 @@ def inner(request, *args, **kwargs):
             # regular users will see a 404.
             if 'version' in str(e):
                 return redirect(reverse('home:home', args=(product,)))
-            elif request.user.is_superuser:
-                url = '%s?product=%s' % (reverse('manage:products'), product)
-                return redirect(url)
```

Comment:
Until then, if the product doesn't exist, this raises an HTTP 500 because the view doesn't exist.

We nixed the products management pages. When we move the products table from alembic/sqlalchemy land to Django land, we could re-add this. I think we shouldn't bother since the new 404 page tells the dev what to do.